includes/Modules/Sign_In_With_Google.php

```diff
@@ -107,4 +109,68 @@ protected function setup_assets() {
 	protected function setup_settings() {
 		return new Settings( $this->options );
 	}
+
+	/**
+	 * Renders the sign in button.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function render_signin_button() {
+		$settings = $this->get_settings()->get();
+
+		$redirect_url = site_url( '/auth/google' );
+
+		if ( substr( site_url(), 0, 5 ) !== 'https' || ! $settings['clientID'] ) {
+			return;
+		}
+
+		// Render the Sign in with Google button and related inline styles.
+		?>
+<!-- <?php echo esc_html__( 'Sign in with Google button added by Site Kit', 'google-site-kit' ); ?> -->
+<style type="text/css">
+.wp-core-ui {
```

Comment:
I know that we have it in IB and I have approved it but now I am trying to understand how much can it be a problem if another plugin also try to mess with the login form styles. 🤔 

Can we achieve the same without using CSS styles?

Reply:
We could do something like this, although it causes layout shift on page load:

```
window.addEventListener("load", () => document.getElementsByClassName('g_id_signin')?.[0] && document.getElementById('login').insertBefore(document.getElementsByClassName('g_id_signin')?.[0], document.getElementById('loginform')));
```